TG.Backend/Controllers/OfferController.cs

```diff
@@ -43,7 +43,7 @@ public async Task<IActionResult> GetOfferById([FromRoute] Guid id)
     }
 
     [HttpPost]
-    [Authorize(AuthenticationSchemes = "Bearer", Roles = "Admin,Employee")]
+    [Authorize(AuthenticationSchemes = "Bearer", Roles = "Admin,Employee,Client")]
```

Comment:
I don't see the necessity to put all three roles here, how about not specyfing roles at all in that scenario? User by default gets added to Client role when registering.